tests/src/compact_geojson.rs

```diff
@@ -0,0 +1,144 @@
+//! Quick-and-dirty implementation of a GeoJSON writer that closely resembles
+//! the formatting used on JS side to simplify diffing result GeoJSONs.
+//!
+//! In case the following upstream issue gets a solution, this module becomes
+//! obsolete:
+//!
+//! https://github.com/serde-rs/serde/issues/1718
+//!
+use geojson::{Feature, Value};
+use serde_json::json;
+
+use std::fs::File;
+use std::io::Write;
+
+
+fn indent_block(indent: i32, s: &str) -> String {
+    let indent = " ".repeat(indent as usize);
+    s.split("\n").enumerate().map(|(i, line)| {
+        if i != 0 {
+            indent.clone() + line
+        } else {
+            line.to_string()
+        }
+    }).collect::<Vec<_>>().join("\n")
+}
+
+trait WriteIndented {
+    fn write_indented<S: AsRef<str>>(self, indent: i32, s: S);
+    fn write_indented_with_substring(self, indent: i32, s: &str);
+}
+
+impl WriteIndented for &mut File {
+    fn write_indented<S: AsRef<str>>(self, indent: i32, s: S) {
+        let indent = " ".repeat(indent as usize);
+        self.write(indent.as_bytes()).expect("Failed to write to file.");
+        self.write(s.as_ref().as_bytes()).expect("Failed to write to file.");
+    }
+
+    fn write_indented_with_substring(self, indent: i32, s: &str) {
```

Comment:
Just noticing this is not used => remove.

Reply:
I think inside a test-case it is just fine to have a helper like this.
Probably makes debugging much easier if the geojson are formatted all the same.

Reply:
The function wasn't even needed for debugging anymore -- I refactored its functionality into the `indent_block` function above, so really just a redundant left over...